src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $main-font-color;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: inherit;
+  }
+
+  &__product-code {
+    color: $secondary-font-color;
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 4px;
+    margin-bottom: 17px;
+  }
+
+  &__review {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__review-line {
+    color: $main-font-color;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    line-height: 18px;
+  }
+
+  &__price-title {
+    color: $secondary-font-color;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__price-amount {
+    color: $main-font-color;
+    font-size: 16px;
+    font-weight: 700;
+  }
```

Comment:
nesting

Reply:
I'm not sure how to do that... "price-title" and "price-amount", for example, are not elements of some "price" block. And we were told not to create an element of element. So I need to rename all that and create a bunch of new blocks in order to do nesting, don't I? If I understand the rule correctly.